python-main.js

```diff
@@ -1156,23 +1156,24 @@ function web_editor(config) {
             } 
         }
 
-        // Show error message
-        $("#flashing-overlay-error").html(
-            '<div>' +
-                '<strong>' + ((err.message === undefined) ? "WebUSB Error" : err.message) + '</strong><br>' + 
-                errorDescription +
-                '<div class="modal-msg-links">' +
+        var errorHTML = '<div>' + ((err.message === undefined) ? "WebUSB Error" : err.message) + '<br >' + 
+                    errorDescription + '<div class="flashing-overlay-buttons">' +
```

Comment:
The buttons div is inside the first div (`<div>Content<div>buttons</div></div>`). Is this on purpose?
It's perfectly fine if that was the intention, the only reason I asked is because maybe it was intended to be `<div>Content</div><div>buttons</div>`?